MIGRATION-TO-SCK.md

```diff
@@ -0,0 +1,326 @@
+# Migration from Splunk Connect for Kubernetes
```

Comment:
This markdown file should be `MIGRATION-FROM-SCK.md`

Reply:
renamed.

Reply:
you said it is renamed, but you haven't pushed that commit.